.github/workflows/full-ci.yml

```diff
@@ -191,11 +191,16 @@ jobs:
         run: |
           echo "TT_CLI_TARANTOOL_PREFIX=${GITHUB_WORKSPACE}/include/" >> $GITHUB_ENV
 
+      - name: Prepare Docker
+        run: |
+          echo "DOCKER_HOST=unix:///${HOME}/.docker/run/docker.sock" >> $GITHUB_ENV
+          docker context use default
+
       - name: Run unit tests
         run: mage unitfull
 
       - name: Integration tests
-        run: mage integrationfull
+        run: TMPDIR=/tmp mage integrationfull
```

Comment:
This change also looks like a workaround for something, please add a comment.

Reply:
Added comment, thank you!